doc/lightning-check.7.txt

```diff
@@ -0,0 +1,37 @@
+LIGHTNING-CHECK(7)
+==================
+:doctype: manpage
+
+NAME
+----
+lightning-check - Command for verifying parameters
+
+SYNOPSIS
+--------
+*check* 'command_to_check' ['parameters']
+
+DESCRIPTION
+-----------
+The *check* RPC command verifies another command's parameters without running it.
+
+The 'command_to_check' is the name of the relevant command.
+
+'parameters' is the command's parameters.
+
+This does not guarantee successful execution of the command in all cases.
+For example, a call to lightning-getroute(7) may still fail to find a route even if
+checking the parameters succeeds.
```

Comment:
This makes me wonder if we should change the command's name to `verify` and not `check`.

Reply:
Meh, "check" is OK.  It would be nicer if we removed more json_tok_tok calls and did proper parsing inline, however.  For example, 'check connect asdfsaf' passes because the parsing of the 'id' parameter happens afterwards.